src/proxy.rs

```diff
@@ -84,7 +86,7 @@ where
                 message = "received json rpc request for",
                 method = method.method
             );
-            if method.method.starts_with("engine_") {
+            if PROXY_METHODS.iter().any(|&m| method.method.starts_with(m)) {
```

Comment:
Does this mean that eth_ methods are not proxied? There are 3 eth_ methods that are also used in rollup-boost.